app.js

```diff
@@ -1,14 +1,15 @@
 
 
 function emailFinder() {
-  let dataset = require('./email-log.json')
+  let dataset = require('./email-log.json');
   const findEmailAddress = dataset.emails.reduce(function (prev, current, index, array) {
```

Comment:
If you're not using index and array, you don't need to add them to your function parameters.